libs/langchain/langchain/chat_models/cohere.py

```diff
@@ -59,6 +59,7 @@ def get_cohere_chat_request(
         ],
         "documents": documents,
         "connectors": maybe_connectors,
+        "prompt_truncation": "AUTO",
```

Comment:
is this something we want to be user configurable?

Reply:
for langchain it doesn't make sense right now. If this is set to off, most usage will probably fail because of context window constraints

Reply:
could you clarify "for langchain it doesn't make sense"? ie why would langchain usage differ from however folks use python client / api directly?

Reply:
it's related to the size and quantity of the input documents and whether we truncate the prompt to do a best effort at fitting within the context window rather than just failing. There is perhaps an argument that AUTO should be the default value on the backend but OFF is considered to be less magic even though request failure is more likely

Reply:
but the above change will apply to all ChatCohere interactions right, not just for retrieval?

Reply:
Ah I see what you mean. I think this will be ignored in normal chat requests but have added a condition to only apply this to the rag use-case! Thanks